wgpu-types/src/lib.rs

```diff
@@ -1066,18 +1424,27 @@ impl Default for TextureAspect {
     }
 }
 
+/// Describes a [`TextureView`].
 #[repr(C)]
 #[derive(Clone, Debug, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 pub struct TextureViewDescriptor<L> {
+    /// Debug label of the texture view. This will show up in graphics debuggers for easy identification.
     pub label: L,
+    /// Format of the texture view. At this time, it must be the same as the underlying format of the texture.
     pub format: TextureFormat,
+    /// The dimension of the texture view. It must be the same as the underlying dimension
```

Comment:
I'm not sure this is correct, and webgpu spec doesn't offer any particular guidance.